Kudu.Core/Deployment/StructuredTextLogger.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Kudu.Core.Tracing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Kudu.Core.Deployment
+{
+    public class StructuredTextLogger : IDetailedLogger
+    {
+        private readonly static object DocumentLock = new object();
```

Comment:
agree that might be too much.. and didn`t notice the namespace.. maybe put some class comments will be good enough, to avoid using this Logger in the wrong way.